assets/css/app.css

```
@@ -452,4 +452,243 @@ body[class*="theme-"] .theme-card:hover {
+.anti-social-intro-compact .text-gray-500 {
+  font-style: italic;
+  font-weight: 400;
+}
+
+/* Focus States for Accessibility */
+.facebook-auth-btn:focus-visible,
+.twitter-auth-btn:focus-visible,
+.social-provider-btn:focus-visible {
+  outline: 2px solid transparent;
+  outline-offset: 2px;
+  ring-offset-2: 2px;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)